DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -195,9 +199,9 @@ void go() {
 
   std::cout << "\nrotations" << std::endl;
 
-  constexpr T a = 0.01;
-  constexpr T ca = std::cos(a);
-  constexpr T sa = std::sin(a);
+  // constexpr T a = 0.01;
+  constexpr T ca = 0.9999500004166653; // std::cos(a);  clang does not support constepxr math functions....
```

Comment:
Just a comment. Clang doesn't support it because C++ standard does not allow it the last time I checked (C++14). IIRC, GCC replaces this to a call to builtin function and that's then they apply constexpr.